lib/scrum/sprint_board.rb

```diff
@@ -16,6 +16,22 @@ def backlog_list_name
       "Sprint Backlog"
     end
 
+    def qa_list_name
+      "QA"
+    end
+
+    def doing_list_name
+      "Doing"
+    end
+
+    def doing_list
+      @board.lists.find { |l| l.name == doing_list_name }
+    end
+
+    def qa_list
+      @board.lists.find { |l| l.name == qa_list_name }
```

Comment:
This is easily done, but ultimately this should be improved upstream.  Well, let's just cache it for now ;)